analyze/algorithm/DiffObjectDetector.cpp

```diff
@@ -36,14 +37,16 @@ namespace am {
 					}
 				}
 				if (nextCheck.size())
-					bfs(changes, nextCheck, object, col);
+					bfs(changes, nextCheck, object, col, minCount);
 
 				return object;
 			}
 
-			std::vector<Pixels> startObjectsSearch(MatrixU16& changes, size_t step, const Column& col)
+			std::vector<Pixels> startObjectsSearch(MatrixU16& changes, am::configuration::Configuration* conf, const Column& col)
 			{
 				std::vector<Pixels> result;
+				const size_t step = conf->PixelStep;
+				const size_t minCountReserve = conf->MinPixelsForObject - 1;

```

Comment:
Allocate vector with a capacity of maximum value of ignoring object,
if capacity will be increased - for sure it is a found object

In case of increasing this value - reallocation of vector happens, but it happens rarely which shall bring faster execution